src/components/pod/pod.test.js

```diff
@@ -13,22 +59,236 @@ const PodWithSoftDelete = () => (
 );
 
 context("Testing Pod component", () => {
-  describe("When softDelete is true", () => {
-    it("renders block with correct background colour", () => {
-      const blockBackgroundColor = "rgb(230, 235, 237)";
+  describe("should render Pod component", () => {
+    it.each([
+      [true, "0px none rgb(0, 0, 0)"],
+      [false, "0px none rgb(0, 0, 0)"],
+    ])(
+      "should check when border is %s the border value is %s for Pod component ",
+      (boolVal, border) => {
+        CypressMountWithProviders(<PodComponent border={boolVal} />);
+        podComponent().should("have.css", "border", border);
```

Comment:
```suggestion
  describe("should render Pod component", () => {
    it.each([
      [true, "1px solid rgb(204, 214, 219)"],
      [false, "0px none rgb(0, 0, 0)"],
    ])(
      "should check when border is %s the border value is %s for Pod component ",
      (boolVal, border) => {
        CypressMountWithProviders(<PodComponent border={boolVal} />);
        podBlock().should("have.css", "border", border);
```
I think you are testing the wrong HTML element here. When `border` is `true` we should not be having these styles.

Reply:
Thanks @Parsium . I'll fix it now.